drivers/input/input_cap12xx.c

```diff
@@ -214,14 +301,22 @@ static int cap12xx_init(const struct device *dev)
 	static struct gpio_dt_spec cap12xx_int_gpio_##index =                                      \
 		GPIO_DT_SPEC_INST_GET(index, int_gpios);))                                         \
 	static const uint16_t cap12xx_input_codes_##index[] = DT_INST_PROP(index, input_codes);    \
+	static const uint8_t cap12xx_signal_guard_##index[] =                                      \
+		DT_INST_PROP(index, signal_guard);                                                 \
+	static const uint8_t cap12xx_calib_sensitivities_##index[] =                               \
+		DT_INST_PROP(index, calib_sensitivity);                                            \
 	static const struct cap12xx_config cap12xx_config_##index = {                              \
 		.i2c = I2C_DT_SPEC_INST_GET(index),                                                \
 		.input_channels = DT_INST_PROP_LEN(index, input_codes),                            \
 		.input_codes = cap12xx_input_codes_##index,                                        \
 		IF_ENABLED(DT_INST_NODE_HAS_PROP(index, int_gpios), (                              \
 				.int_gpio = &cap12xx_int_gpio_##index,))                           \
 		.repeat = DT_INST_PROP(index, repeat),                                             \
-		.poll_interval_ms = DT_INST_PROP_OR(index, poll_interval_ms, 10)};                 \
+		.poll_interval_ms = DT_INST_PROP_OR(index, poll_interval_ms, 10),                  \
+		.analog_gain = DT_INST_PROP_OR(index, sensor_gain, 0),                             \
```

Comment:
it's usually a good idea to match the config field name with the corresponding property, either `analog_gain` and `analog-gain` or `sensor_gain` and `sensor-gain`, pick one (the one in the datasheet I guess)